### ISSUE TITLE
IONcore Developer Fund Signatories/Management/Implementation

### ISSUE THREAD

**Issue by korvasc** [ioncoincore/ion#14]
Need to know how much ION IONcore have available. The amount has a fundamental impact on how this project moves forward, so it's something that as a priority needs resolution. 

Actions

1. Signatories Agree
2. LordDarkHelmet liaises with Huey to confirm amount ION to be transferred to IONcore 
3. LordDarkHelmet to implement IONcore developer fund
 
Developer Fund Signatories 
**Huey** 
Ionomy Visionary
Ionomy aware of how IONcore spends the ION.
Avoids duplication of effort with whatever Ionomy is working on
Encourages people in Ionomy to contribute when they see leaders collaborate

**TOR**
Ion Visionary
Has the expertise to evolve ION into a ground-breaking world-leading cryptocurrency
Has an intimate understanding of the development process
Has a history of turning ideas into reality.
Saved ion

**Korvas**
A respected member of the community 
20+ years of success in Sales, Marketing & Business development
    10 years Intel
    2 years VideoLogic/Imagination Technologies
    12 years Creative Labs
 Involved in cryptocurrency since  2013

IONcore Financial Controller 
**LordDarkhelmet**
A respected member of the community
Has the expertise & trust to be able to set up and manage IONcore Developer fund

Additional comments
Projects reviewed on a case by case basis.

IONcore will be able to define and implement a bounty list.

Milestone End Nov 2020

**Comment by korvasc** [comment]
As I put this together I off course agree to be a signatory

**Comment by LordDarkHelmet** [comment]
Step 1: Understand the current financial position. What are our ongoing costs, liabilities and burn rate. 
Step 2: Break down future spend
Step 3: Understand Objectives. (What are we trying to accomplish, what is our mission, what are our priorities, how funds are distributed, ...)
Step 4: Come up with procedures for general practices
Step 5: Ensure proper reporting and transparency 
Step 6: Continuous feedback loop to ensure we are meeting our objectives

**Comment by adammatlack** [comment]
I agree to be a signer on this. Currently, the step 1 answer is at a minimum we are ready to move a bit over 3.1mm ION into a multisig for core to control and leverage.  I need to do more accounting to make sure it is not missing any and its the right amount.

**Comment by korvasc** [comment]
LordDarkHelmet I like the structure although I recommend step 3 be step 2 and vice versa. This will help make sure future spend aligns to the defined objective/s

**Comment by LordDarkHelmet** [comment]
Sometimes people get 2 and 3 confused. 

Step 1 and 2 are there to create a baseline. Step 1 establishes current status and commitments, Step 2 looks at future commitments and allocations that are currently known and agreed upon.

Step 3 then examines what we would like to do. Any action in step 3 is a delta from our baseline established in step 1 and 2. Each delta needs to be reviewed and budgeted accordingly as they are changes to the established plan. 

Once the action in step 3 is approved, we need to make sure we are following procedures, or come up with new ones. This is step 4. 

Step 5 is about execution. Basically, we need to be transparent and report what is going on.

Step 6 is our loop, we review current progress and make changes to the plan as needed (Step 3)

Perhaps a flowchart is better than text:

![image](https://user-images.githubusercontent.com/27186038/98446641-e584fa00-20ec-11eb-88fb-b3948101df5e.png)

**Comment by cryptolinux** [comment]
I agree to the proposal.

I find it really good that @LordDarkHelmet created some scheme on which we can elaborate, as on one side we can create everything from scratch, on other side we already have a lot of it, maybe good start for step 1 would be evaluation of what we already have.

**Comment by korvasc** [comment]
Great this issue is complete so in summary

I agreed to be a signatory
Huey agreed to be a signatory
Tor agreed to be a signatory
LordDarkHelmet agreed to be Ioncore Financial Controller

If anyone has any objections let me know within the next 24/48 hours otherwise I will close this issue

**Comment by korvasc** [comment]
Nov 16 2020 - Meeting

Attendee's     Huey, Cryptolinux, Korvas

Process of ION transfer from Ionomy to Ioncore has begun, funds to be stored in a Multi-Sig Wallet

Cryptolinux will create a Multi-Sig wallet for the ION

Everyone agreed to Cryptolinux recommendation that we increase the number of signatories from 3 to 5

1 - Huey                                    Public key - submitted
2 - Cryptolinux                          Public key - submitted
3 - Korvas                                  Public key - submitted
**4 - LordDarkHelmet                  Please let Cryptolinux know your Public key**
5 - To be confirmed                  

Also discussed the need from an additional wallet that would hold funds to support the day to day IONcore running costs 

Let me know if I've missed something

End

**Comment by korvasc** [comment]
Update Dec 14

Crytolinux setup an MS wallet which we were able to successfully test. The only problem is I let slip my private key so another wallet will need to be created. 

Dark will set this up, we will test then I will publish details for community review

**Comment by korvasc** [comment]
Financial Controller                     
LordDarkHelmet

Signatories                                  
Cryptolinux 
adammatlack
korvasc

To be reviewed in approx 12mths
